e2e/pages/Workspaces.js

```diff
@@ -1,11 +1,17 @@
 const Page = require('./Page');
+let CommonActions = require('../utils/CommonActions.js');
 class Workspaces extends Page {
+
+    constructor(){
+        super();
+        this.workspaceContext = '.raw_context_name';
+    }
     open() {
-        super.open('/workspaces');
+        super.open('/n/workspaces/${workspaceId}');
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace the quotes (') with back-ticks (`). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS3786)